test/packages/good/data_stream/foo/manifest.yml

```diff
@@ -1,5 +1,6 @@
 title: Nginx access logs
 type: logs
+allow_routing: true
```

Comment:
If the field `allow_routing` isn't supposed to be mandatory, I guess that you can leave only for the `good` package and drop it from other places. You don't need to verify the same feature multiple times.